### ISSUE TITLE
FEAT: add more context options to table tabs

### ISSUE THREAD

**Issue by Zerotask** [beekeeper-studio/beekeeper-studio#786]
**Is your feature request related to a problem? Please describe.**
If you right click on a table tab, you have a different context menu than right-clicking on the table in the `Entities` section.

For example if you're looking at the table data and you want to inspect the table structure, you can't simply right-click on the table tab. You have to right-click on the table name in the `Entities` section.

**Describe the solution you'd like**
Add the same context options to a table tab

**Additional context**
Current table tab context menu:
![image](https://user-images.githubusercontent.com/20150243/131368086-59271e62-c41d-4637-8aa7-caf15b45e1c2.png)

Context menu from the `Entities` section
![image](https://user-images.githubusercontent.com/20150243/131368152-ca60eda9-b2ab-420f-b786-a15d66bf32f9.png)

**Comment by txigreman** [comment]
You have a button below the data table to jump to the structure:

![imagen](https://user-images.githubusercontent.com/778778/132642042-fb31af3f-2ea9-4301-ae34-71443189c765.png)

**Comment by Zerotask** [comment]
Yes, I know that.

> (or click in the footer bar on Structure which can be overlooked)

**Comment by rathboma** [comment]
Yeah this is interesting as we'd always thought of the 'tab headers' as their own thing. But what you say does make sense.

I'll mark it. There's probably a way to centralize the two context menu implementations to do this nicely.
